.php_cs.dist

```diff
@@ -9,11 +9,13 @@ $finder = PhpCsFixer\Finder::create()
 ;
 
 return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
```

Comment:
Need to allow risky fixers as the detection of PHPUnit test classes could fail, and we could end up fixing the visibility of `setUp()` and `tearDown()` methods in unrelated classes.